website/src/lib/constants.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  useCreateAssistantReply,
+  useCreateInitialPrompt,
+  useCreatePrompterReply,
+} from "src/hooks/tasks/useCreateReply";
+import { useGenericTaskAPI } from "src/hooks/tasks/useGenericTaskAPI";
+import {
+  useLabelAssistantReplyTask,
+  useLabelInitialPromptTask,
+  useLabelPrompterReplyTask,
+} from "src/hooks/tasks/useLabelingTask";
+import {
+  useRankAssistantRepliesTask,
+  useRankInitialPromptsTask,
+  useRankPrompterRepliesTask,
+} from "src/hooks/tasks/useRankReplies";
+import { TaskType } from "src/types/Task";
+
+export const ERROR_CODES = {
+  TASK_REQUESTED_TYPE_NOT_AVAILABLE: 1006,
+  TASK_INVALID_REQUEST_TYPE: 1000,
+  TASK_ACK_FAILED: 1001,
+  TASK_NACK_FAILED: 1002,
+  TASK_INVALID_RESPONSE_TYPE: 1003,
+  TASK_INTERACTION_REQUEST_FAILED: 1004,
+  TASK_GENERATION_FAILED: 1005,
+  TASK_AVAILABILITY_QUERY_FAILED: 1007,
+  TASK_MESSAGE_TOO_LONG: 1008,
+};
+
+export const apiHooksByType = {
```

Comment:
could you maybe add types to this? something like:

```ts
const apiHooksByType: { [taskType in TaskType]: (...args: any[]) => any } = ...
```
or even something better defined if you can.

Reply:
Yes, no problem.